lib/formatters/stylish.js

```diff
@@ -73,14 +77,22 @@ module.exports = function(results) {
         ).split("\n").map(el => el.replace(/(\d+)\s+(\d+)/, (m, p1, p2) => chalk.dim(`${p1}:${p2}`))).join("\n")}\n\n`;
     });
 
-    const total = errors + warnings;
+    const total = errorCount + warningCount;
 
     if (total > 0) {
         output += chalk[summaryColor].bold([
             "\u2716 ", total, pluralize(" problem", total),
-            " (", errors, pluralize(" error", errors), ", ",
-            warnings, pluralize(" warning", warnings), ")\n"
+            " (", errorCount, pluralize(" error", errorCount), ", ",
+            warningCount, pluralize(" warning", warningCount), ")\n"
         ].join(""));
+
+        if (fixableErrorCount > 0 || fixableWarningCount > 0) {
+            output += chalk[summaryColor].bold([
+                "  ", fixableErrorCount, pluralize(" error", fixableErrorCount), ", ",
+                fixableWarningCount, pluralize(" warning", fixableWarningCount),
+                " can potentially be fixed by running eslint with the `--fix` option.\n"
```

Comment:
Current output:

![](https://i.gyazo.com/00ede1770fb3e2a1baed1ed2915db425.png)

Compared to the previous output, this seems a bit verbose to me. Can we reword it? For example:

> 2 problems (1 error, 1 warning), 1 problem potentially fixable with the \`--fix\` option

Since the `--fix` flag tries to fix both errors and warnings anyway, I don't think it's worth displaying fixable counts for each type of problem.

I'm open to bikeshedding on this. The reason I'm concerned about keeping the message short is because this gets printed for a large percentage of reports, so the output will need to be read quite a lot. I think it will be better for UX if we can keep the message as simple as possible while still conveying the same information.